lib/renderer/web-view/web-view-attributes.ts

```diff
@@ -186,7 +186,10 @@ export class SrcAttribute extends WebViewAttribute {
       opts.userAgent = useragent;
     }
 
-    (this.webViewImpl.webviewNode as Electron.WebviewTag).loadURL(this.getValue(), opts);
+    (this.webViewImpl.webviewNode as Electron.WebviewTag).loadURL(this.getValue(), opts)
+      .catch(err => {
+        console.error('Unexpected error while loading URL', err);
```

Comment:
Wouldn't it be a much better API if we returned the promise, so that callers can handle the error in any way that seems appropriate instead of printing it to stderr?

Reply:
Return promise should change `parse` function and all callers to be async function. 

In this case the callers is webview itself not developers, so I think it is enough to just print the error. If you have seen the call stack, you will find that developers have no way to handle this error.

Reply:
Hmm, makes sense. I guess that's the reason why there are similar logging statements in this file like https://github.com/electron/electron/blob/75f9573e53338e15af7c7ccd07a529332847ea2c/lib/renderer/web-view/web-view-attributes.ts#L87